pallets/ddc-payouts/src/lib.rs

```diff
@@ -60,10 +60,10 @@ pub struct NodeUsage {
 /// Stores reward in tokens(units) of node provider as per NodeUsage
 #[derive(PartialEq, Encode, Decode, RuntimeDebug, TypeInfo, Default, Clone)]
 pub struct NodeReward {
-	pub transfer: u128, // tokens for transferred_bytes
-	pub storage: u128,  // tokens for stored_bytes
-	pub puts: u128,     // tokens for number_of_puts
-	pub gets: u128,     // tokens for number_of_gets
+	pub transfer: u64, // tokens for transferred_bytes
```

Comment:
If the `transfer` and `storage` fields now represent the number of bytes as `u64`, I think we should write a different comment that explicitly says that it is a number of bytes but not tokens.
For tokens, we should always use `u128` as it is defined for the Balance type in runtime

Reply:
So, let's just write a better comment to bring more clarity on why there is `u64` for `transfer` and `storage`, and `u128` for `puts` and `gets`. Currently, all types have the notion of `token` in their comments,  it is ambiguous as tokens should always be of `u128` type